packages/nextjs-kit/src/components/header.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Link from 'next/link';
+import { HeaderData } from '../types/index';
+
+interface HeaderProps {
+  navItems: HeaderData[];
+}
+
+/**
+ * This is a Header component.
+ *
+ * @param {HeaderProps} navItems holds the title, href, and alignment of each route to be displayed
+ * @returns {React.JSX.Element}
```

Comment:
```suggestion
 * @returns {JSX.Element} A header with a navigation menu
```
JSX is its own class. Also add a description of the component that is being returned